### DISCUSSION TITLE
Conference NTTS 2025 (Brussels, March 11-13) – Who is attending, any GridViz involved?

### DISCUSSION THREAD

**Discussion by wahlatlas** [eurostat/gridviz#140] [General]
*New Techniques and Technologies for Statistics (NTTS) is an international biennial scientific conference series, organized by Eurostat, on new techniques and methods for official statistics, and the impact of new technologies on statistical collection, production and dissemination systems.*  

Find everything including DRAFT OUTLINE PROGRAMME at  
https://cros.ec.europa.eu/ntts2025  
  
Today acceptance mails to submitters where received.  

It looks to me that datavis and mapping don't play a central role in the upcoming edition but GridViz might play a role here and there nevertheless.

**Comment by jgaffuri** [comment]
Hi Michael !
I plan to go and hope to meet you there ! We do not plan a presentation on gridviz - we already had one two years ago.
But there will be several presentations on eurostat activities where gridviz was used. We also plan to show that https://e.pcloud.link/publink/show?code=kZKv8PZEjeMOFVT0OuUi9NBg0D7LJ1yTNO7 during the poster session.

**Reply by wahlatlas** [reply]
It looks like datavis will be mainly in the poster presentations

**Comment by jgaffuri** [comment]
Maybe you plan to present your work on census 2021 ? Such 100m resolution gridded statistics must be shown !

**Reply by wahlatlas** [reply]
It will be on census 2022 but mostly not about maps but value distributions and skewness instead. Should nevertheless be a good opportunity to catch up. Hopefully by then I will be fluent in v3 of gridviz. As for the 100m resolutions, I'll post something here.
